_api-reference/snapshots/get-snapshot-status.md

```diff
@@ -29,9 +29,9 @@ Three request variants provide flexibility:
 
 * `GET _snapshot/_status` returns the status of all currently running snapshots in all repositories.
 
-* `GET _snapshot/<repository>/_status` returns the status of only currently running snapshots in the specified repository. This is the preferred variant.
+* `GET _snapshot/<repository>/_status` returns all currently running snapshots in the specified repository. This is the preferred variant.
```

Comment:
Why is this the preferred variant of this command?

Reply:
@harishbhakuni  - do you know why this is the recommended option?